addon/routes/building/new.js

```diff
@@ -18,7 +18,7 @@ export default class BuildingNewRoute extends BuildingFormRoute {
 
   async model() {
     const model = {
-      instanceId: this.modelFor("application").id,
+      instanceId: this.modelFor("application")?.id,
```

Comment:
Will the `instanceId` ever be undefined here?

Reply:
maybe not but i added the optional chaining everywhere and it never hurts to be a bit pessimistic